tests/Integration/Subscriptions/EchoBroadcaster/AuthorizeRequestsTest.php

```diff
@@ -27,6 +27,23 @@ public function testEchoClientAuthorizesSuccessfully(): void
     {
         $response = $this->querySubscription();
 
+        $channel = $response->json('extensions.lighthouse_subscriptions.channels.taskUpdated');
```

Comment:
```suggestion
        $channel = $response->json('extensions.lighthouse_subscriptions.channel');
```

Reply:
I don't understand it. Do you mean to update the variable name because of the purpose? It's been all over the class, should I change each of them as well or just the test I've added?

Reply:
I changed existing uses https://github.com/nuwave/lighthouse/pull/1735, you can just apply the suggestion,.